src/kOS/Utilities/SteeringHelper.cs

```diff
@@ -162,31 +167,65 @@ public static Vector3d GetTorque(Vessel vessel, float thrust)
                 var pod = part as CommandPod;
```

Comment:
As the code seems to be working on crafts that only have reaction wheels in the command pod, I guess that they are accessed later on, when accessing the ModuleReactionWheel part modules. I guess one can safely remove this obsolete code.